Sync dev-es branch with main

electrocucaracha opened the pull request:
### Describe your changes
Sync the dev-es branch with the main branch

### Related issue number or link (ex: `resolves #issue-number`)
NA

### Checklist before opening this PR (put `x` in the checkboxes)
- [x] This PR does not contain plagiarism
  - don’t copy other people’s work unless you are quoting and contributing it to them.
- [x] I have signed off on all commits 
  - [signing off](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) (ex: `git commit -s`) is to affirm that commits comply [DCO](https://wiki.linuxfoundation.org/dco). If you are working locally, you could add an alias to your `gitconfig` by running `git config --global alias.ci "commit -s"`.

jihoon-seo commented:
Hi @electrocucaracha, thank you for your work.
However,
- to push the commits in `main` branch into `dev-es` branch, you can achieve it by just opening a PR that merges `cncf:main` into `cncf:dev-es`, as long as a merge conflict does not occur.
  To open a PR that merges `cncf:main` into `cncf:dev-es`, you can use this link: https://github.com/cncf/glossary/compare/dev-es...main
  You can see that currently there are only 3 commits that will be pushed from `main` to `dev-es`.
- It seems that you used `git rebase` command, since all the commits in this PR say that they are committed by you @electrocucaracha.
  ![image](https://user-images.githubusercontent.com/46767780/228393417-b315ded1-4010-4843-b3b3-1cf24209c6bd.png)
  In the process of rebasing, the hashes of commits from `main` changed, so GitHub could not detect that the commits in this PR are the same ones in `main`, and displayed as 'commits that will be pushed by this PR'.
  The command that we need to use to achieve this would be `git merge`, not `git rebase`.

@electrocucaracha could you open a new PR that merges `cncf:main` into `cncf:dev-es` by using the link https://github.com/cncf/glossary/compare/dev-es...main ?
PTAL, and thank you!

electrocucaracha commented:
> @electrocucaracha could you open a new PR that merges `cncf:main` into `cncf:dev-es` by using the link [dev-es...main](https://github.com/cncf/glossary/compare/dev-es...main) ? PTAL, and thank you!

@jihoon-seo thanks for the clarification, as you said I used `git rebase` instruction, and also I fixed some commit messages and I squashed it some others like [these](https://github.com/cncf/glossary/pull/1754). Anyway, I'll close this and open a new one using the link.
